dashboard/api.py

```diff
@@ -350,6 +284,7 @@ def format_courserun_for_dashboard(course_run, status_for_user, certificate=None
         status_for_user (str): a string representing the status of a course for the user
         certificate (Certificate): an object representing the
             certificate of the user for this run
+        position (int): The position of the course run within the list
```

Comment:
gotcha - well, I'm fine with leaving it for now, but if we're really not using it for anything we should probs delete it.